crates/telio-traversal/src/upgrade_sync.rs

```diff
@@ -615,7 +616,7 @@ mod tests {
             upg_decision_us,
             our_public_key,
             Arc::new(ConnectivityDataAggregator::new(
-                None,
+                AggregatorConfig::default(),
                 wg.clone(),
                 our_public_key,
             )),
```

Comment:
Waait, why it creates a new instance of aggregator?

Reply:
`AggregatorConfig::default()` is `false` for both nat-monitoring and derp. Not sure about why it is being created. Note, these are tests.

Reply:
Yeah, right